### PR TITLE
transform boxes

### PR DESCRIPTION
**Pull request by dvd42**

### CODE REVIEW

File: kornia/geometry/linalg.py [hunk 1]
```diff
@@ -213,6 +214,53 @@ def transform_points(trans_01: torch.Tensor,
...
+    if not torch.is_tensor(trans_mat):
+        raise TypeError(f"Tranformation matrix type is not a torch.Tensor. Got {type(trans_mat)}")
+
+    if not isinstance(mode, str):
+        raise TypeError(f"Mode must be a string. Got {type(mode)}")
+
+    if mode not in ("xyxy", "xywh"):
+        raise ValueError(f"Mode must be one of 'xyxy', 'xywh'. Got {mode}")
+
+    boxes = boxes.unsqueeze(0)
```
Comment (edgarriba) [inline kornia/geometry/linalg.py]: What's the boxes shape then ? I would assume Nx4 or BxNx4

Comment (priba) [reply kornia/geometry/linalg.py]: Agree, I would ask for Nx4 or BxNx4 as you usually have several boxes per image.

File: test/geometry/test_linalg.py [hunk 2]
```diff
@@ -108,6 +108,105 @@ def op_script(transform, points):
...
 
+class TestTransformBoxes:
+
+    def test_transform_boxes(self, device):
+
+        boxes = torch.Tensor([[139.2640, 103.0150, 397.3120, 410.5225]]).to(device)
+
+        expected = torch.Tensor([372.7360, 103.0150, 114.6880, 410.5225]).to(device)
+
+        trans_mat = torch.Tensor([[[-1., 0., 512.],
```
Comment (edgarriba) [inline test/geometry/test_linalg.py]: use `torch.tensor` constructor

File: test/geometry/test_linalg.py [hunk 3]
```diff
@@ -108,6 +108,105 @@ def op_script(transform, points):
...
+                                   [0., 1., 0.],
+                                   [0., 0., 1.]]]).to(device)
+
+        out = kornia.transform_boxes(trans_mat, boxes)
+        print(out)
```
Comment (edgarriba) [inline test/geometry/test_linalg.py]: remove print

**Pull request merged**